test/integration/nearcache/LostInvalidationsTest.js

```diff
@@ -91,24 +90,30 @@ describe('LostInvalidationTest', function () {
         const updatedval = 'updatedval';
 
         const map = await client.getMap(mapName);
-        await TestUtil.promiseWaitMilliseconds(100);
-        const invalidationHandlerStub = blockInvalidationEvents(client, map, 1);
+        // Wait for the near cache to be ready. This also means that invalidation listener is registered.
+        await map.nearCache.ready.promise;
+        // One event comes after the first put and the other comes after the second put of the modifying client.
+        const invalidationHandlerStub = blockInvalidationEvents(client, map, 2);
         const mp = await modifyingClient.getMap(mapName);
         await mp.put(key, value);
         await map.get(key);
         await mp.put(key, updatedval);
-        await TestUtil.promiseWaitMilliseconds(1000);
+        // Wait till invalidation events to come.
+        await invalidationHandlerStub.notificationPromise;
```

Comment:
How does this test pass? You didn't rename `notificationHandler` to `notificationPromise` above